.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
+
+on:
```

Comment:
We will want to update this to run on PRs and our major release lines I think.

Reply:
Yea, it should run on all branch pushes and on prs

Reply:
Easily enough done.  We can modify the triggers to be whatever you would like.  I'll add in a trigger for PRs to master.

Reply:
It should do prs to any branch. For example we make prs all the time to the release branches.

Reply:
I've made this change.